### PR TITLE
Small improvements for basic slides

### PR DESCRIPTION
**Pull request by bernhardmgruber**
Here are some suggestions which I collected while reading through the basic course slides again.

Fixes: #218

### CODE REVIEW

File: talk/morelanguage/lambda.tex [hunk 1]
```diff
@@ -341,3 +342,4 @@
   \end{exampleblock}
 
 \end{frame}
+\end{advanced}
```
Comment (sponce) [inline talk/morelanguage/lambda.tex:345]: This change should be dropped after rebasing on master. This has already been done in another MR (and more slides have been moved to advanced actually)

**Review by sponce — APPROVED** [review]

File: talk/morelanguage/lambda.tex (hunk 1 above)
Comment (bernhardmgruber) [reply talk/morelanguage/lambda.tex:347]: Right, I saw the other PR.
